sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -0,0 +1,133 @@
+              {{ item.restrictionType }}
+            </template>
+            <template #[`item.restrictionCode`]="{ item }">
+              {{ item.restrictionCode }}
+            </template>
+            <template #[`item.isActive`]="{ item }">
+              <status-chip-bypass :is-active="item.isActive" />
+            </template>
+            <template #[`item.id`]>
+              <v-btn
+                color="primary"
+                variant="text"
+                class="text-decoration-underline"
+                prepend-icon="fa:far fa-file-alt"
+              >
+                View Details</v-btn
+              >
+            </template>
+            <template #[`item.isBypassActive`]="{ item }">
+              <v-btn
+                :color="getRemoveBypassColor(item.isBypassActive)"
+                :disabled="!item.isBypassActive"
+              >
+                {{ getRemoveBypassLabel(item.isBypassActive) }}</v-btn
+              >
+            </template>
+          </v-data-table>
+        </toggle-content>
+      </content-group>
+    </v-container>
+  </v-card>
+</template>
+<script lang="ts">
+import {
+  ApplicationStatus,
+  DEFAULT_PAGE_LIMIT,
+  PAGINATION_LIST,
+  ApplicationRestrictionManagementHeaders,
+} from "@/types";
+import { ref, onMounted, defineComponent, PropType } from "vue";
+import { ApplicationRestrictionBypassHistoryAPIOutDTO } from "@/services/http/dto/ApplicationRestrictionBypass.dto";
+import StatusChipBypass from "@/components/generic/StatusChipBypass.vue";
+import { ApplicationRestrictionBypassService } from "@/services/ApplicationRestrictionBypassService";
+
+export default defineComponent({
+  components: {
+    StatusChipBypass,
+  },
+  props: {
+    applicationId: {
+      type: Number,
+      required: true,
+    },
+    applicationStatus: {
+      type: String as PropType<ApplicationStatus>,
+      required: true,
+    },
+  },
+  setup(props) {
+    const isDraftApplication =
+      props.applicationStatus === ApplicationStatus.Draft;
+    const bypassedRestrictions = ref({
+      bypasses: [],
+    } as ApplicationRestrictionBypassHistoryAPIOutDTO);
+    onMounted(async () => {
+      if (!isDraftApplication) {
+        bypassedRestrictions.value =
+          await ApplicationRestrictionBypassService.shared.getApplicationRestrictionBypassesHistory(
+            props.applicationId,
+          );
+      }
+    });
+
+    const getRemoveBypassLabel = (isActive: boolean): string => {
+      return isActive ? "Remove Bypass" : "Bypass Removed";
+    };
+
+    const getRemoveBypassColor = (isBypassActive: boolean): string => {
+      return isBypassActive ? "primary" : "secondary";
+    };
+
+    const getToggleContentMessage = (): string => {
+      if (props.applicationStatus === ApplicationStatus.Draft) {
+        return "Bypass restriction disabled for draft applications.";
+      } else if (!bypassedRestrictions.value.bypasses.length) {
+        return "No bypassed restrictions found.";
```

Comment:
There is no requirement to have specific toggle content message for draft and  no data. 
` "No bypassed restrictions found."` should be good enough.

Reply:
This method handles the toggle content message for when the application is a draft application and when the there is bypass records. When the application is a draft application, it displays "Bypass restriction disabled for draft applications." As per our discussion in Teams, I think you meant that we should change the text to "No assessments found." for the message for draft applications?

Reply:
I meant, either the application is draft or there is no bypasses display one generic message in the togglt that is `No bypassed restrictions found.` and there is no need to apply logic to have secondary message like `Bypass restriction disabled for draft applications`. 

```vue
<toggle-content
        :toggled="!bypassedRestrictions.bypasses.length || isDraftApplication"
        message="No bypassed restrictions found"
      >
```

Reply:
Updated. We won't be worried about `isDraftApplication` at this point and the API query is filtering out draft applications.